servers/v3-proxy-api/src/graph/toGraphQL.spec.ts

```diff
@@ -3,25 +3,125 @@ import { setSaveInputsFromGetCall } from './toGraphQL';
 import {
   SavedItemsSortBy,
   SavedItemsSortOrder,
-  SavedItemStatusFilter,
 } from '../generated/graphql/types';
+import { defaultQuery } from '../test/fixtures';
 
 describe('toGraphQL', () => {
-  //todo: refactor this test as you refactor the method
   it('should map saves input', () => {
     const expected = {
       pagination: {
         limit: 30,
         offset: 0,
       },
       sort: {
-        sortBy: SavedItemsSortBy.UpdatedAt,
-        sortOrder: SavedItemsSortOrder.Asc,
-      },
-      filter: {
-        status: SavedItemStatusFilter.Unread,
+        sortBy: SavedItemsSortBy.CreatedAt,
+        sortOrder: SavedItemsSortOrder.Desc,
       },
+      filter: {},
     };
-    expect(setSaveInputsFromGetCall('sample-rest-params')).toEqual(expected);
+    expect(setSaveInputsFromGetCall(defaultQuery)).toEqual(expected);
+  });
+  describe('filter object', () => {
+    // Need to see how Express handles this
+    it.todo('handles duplicate query parameters');
+    it.each([
+      {
+        params: { since: 123456 },
+        expected: { filter: { updatedSince: 123456 } },
+      },
+      {
+        params: { state: 'all' as const },
```

Comment:
wouldn't this come in as a string? Why the cast to `const`?

Reply:
The explicit union type e.g. `'query' | 'all'` doesn't typecheck the string properly unless you cast it to const